tsconfig.json

```diff
@@ -27,9 +27,9 @@
         // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
         /* Modules */
-        "module": "NodeNext" /* Specify what module code is generated. */,
+        "module": "ES2022" /* Specify what module code is generated. */,
         // "rootDir": "./",                                  /* Specify the root folder within your source files. */
-        "moduleResolution": "NodeNext" /* Specify how TypeScript looks up a file from a given module specifier. */,
+        "moduleResolution": "bundler" /* Specify how TypeScript looks up a file from a given module specifier. */,
```

Comment:
Rationale for this change: https://github.com/GoogleChromeLabs/jsbi/issues/96#issuecomment-2480828102